src/groups/bmq/bmqimp/bmqimp_brokersession.cpp

```diff
@@ -1350,8 +1381,25 @@ void BrokerSession::QueueFsm::actionInitiateQueueResume(
         context);
 }
 
+void BrokerSession::QueueFsm::logOperationTime(const int   queueId,
```

Comment:
I am curious if logging `queueId` here is a good way.
We might log queue uri instead, and it's easier to search in logs.
Queue uri for the same queue will always be the same across different launches and even across client/cluster, but `queueId` might be different.

Reply:
Inside `BrokerSession` class `queueId` is used for logging states transitions, but agree with you that for operation time `queueUri` is more readable/searchable. Changed to `queueUri`.